### DISCUSSION TITLE
Delete old generated

### DISCUSSION THREAD

**Discussion by Samuel-Morgan-Tyghe** [kubb-labs/kubb#1080] [Ideas]
So I have a script on the FE to delete the whole generated folder. so each Kubb run generates everything from scratch.

This is needed to clean out all deleted (from the contract)

**Comment by Samuel-Morgan-Tyghe** [comment]
perhaps a CLI param? `--clean`

**Reply by stijnvanhulle** [reply]
https://kubb.dev/config/output#output-clean
